assets/src/application/ui/layout/entityList/filterBar/EntityListFilterBar.tsx

```diff
@@ -1,34 +1,27 @@
-/**
- * External imports
- */
 import * as React from 'react';
 import { AppstoreFilled, FilterOutlined, TableOutlined } from '@ant-design/icons';
 import { __ } from '@wordpress/i18n';
 import EspressoButton from '../../../input/EspressoButton';
 
-/**
- * Internal dependencies
- */
-import useEntityListFilterState from './useEntityListFilterState';
 import Collapsible from './Collapsible';
 import './style.css';
 
 /**
  * EntityListFilterBar
  * a group of inputs for controlling how a list of entities is displayed
  */
-const EntityListFilterBar = ({ listId, entityFilters = null }) => {
+const EntityListFilterBar = ({ entityFilters = null, filterState, listId }: any) => {
```

Comment:
can this not be typed more strictly?

Reply:
Improved here: https://github.com/eventespresso/event-espresso-core/pull/2431/commits/f2d2dcb3731f859181f099a17f920bc25e541c26